WCSAxes: fix get_axislabel() to return the default label if no label has been explicitly provided (#17175)
astropy/astropy: docs/changes/visualization/17175.bugfix.rst

> Fixed a bug that caused ``CoordinateHelper.get_axislabel()`` to not return
> anything instead of the default label if no label has been explicitly

Reviewer: ```suggestion
Fixed a bug that caused ``CoordinateHelper.get_axislabel()`` to return an
empty string instead of the default label if no label has been explicitly
```
